.github/actions/benchmark/action.yml

```diff
@@ -0,0 +1,147 @@
+name: 'Benchmark'
+description: 'Run benchmarks for Asterinas'
+inputs:
+  task:
+    description: 'Task to run (benchmark, result)'
+    required: true
+  architecture:
+    description: 'Architecture to benchmark (x86-64, tdx)'
```

Comment:
Since Intel TDX is not a CPU architecture, I suggest using the term "platform" instead.

The `benchmark/README.md` should also use the term platform consistently.

```md
are available for all tier-1 supported platforms.
```